homework/Anastasia_Kichaeva/Hw5_Task1/src/main/java/acya/ki/bean/CounterBean.java

```diff
@@ -0,0 +1,65 @@
+package acya.ki.bean;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class CounterBean implements Serializable {
+    private int value;
+    private int step;
+
+    public CounterBean() {
+        this.value = 0;
+        this.step = 1;
+    }
+
+    public CounterBean(int value, int step) {
+        this.value = value;
+        this.step = step;
```

Comment:
You can put any integer in value and step through constructor
Add if() statement